tachyon/math/polynomials/univariate/radix2_evaluation_domain.h

```diff
@@ -267,6 +350,160 @@ class Radix2EvaluationDomain : public UnivariateEvaluationDomain<F, MaxDegree> {
     }
   }
 
+  // This can be used as the first half of a parallelized butterfly network.
+  CONSTEXPR_IF_NOT_OPENMP void ParDitLayer(RowMajorMatrix<F>& mat, size_t mid,
```

Comment:
This doesn't match our naming convention. We try to name methods as verbs, but this is an exception. Also, we add a `Parallel` suffix for the parallel versioned functions.

Reply:
Could you give me an example from within our codebase? I can only seem to find `Parallel` being used as a prefix or the middle of function names